docs/api-reference/cloud-cli.md

```diff
@@ -123,6 +123,17 @@ This command destroys a previously-provisioned Postgres database instance.
 
 ---
 
+### `npx dbos-cloud db connect`
```

Comment:
Sounds like after running this command, the password would be stored in plain text in the dbos-config file... may want to add a warning to not commit this modified file to source control.